client/src/state/store.ts

```diff
@@ -31,6 +33,7 @@ export const store = configureStore({
 store.subscribe(
   debounce(() => {
     saveState(store.getState()[progressSlice.name]);
+    savePreferences(store.getState()[preferencesSlice.name]);
```

Comment:
The reason I would like an option "save my settings (to local storage)" is that I think european GDPRs might somewhat apply here, similarly to using cookies. For anything not "strictly necessary for the functionality of the website" you need the user's confirmation.

That could just be a third option in the Preferences.

Reply:
I wrote a new issue for this as I want to merge your changes:
- [ ] #172

Reply:
OK I will continue to follow up on this issues.